mibidata/mibi_image.py

```diff
@@ -56,14 +56,17 @@ class MibiImage():
             peak locations after mass recalibration.
         check_reg: Whether or not the maximum shift between depths is higher
             than a threshold.
+        filename: The name of the Run XML file which corresponds to the run
```

Comment:
We're using JSON files for MIBIscope 1 :) It's also a convention that the filename corresponds to the run name, but it doesn't need to be that way.

Reply:
Fixed. It now reads `filename: The name of the file with the run metadata.`